src/main/java/io/cdap/plugin/servicenow/ServiceNowBaseConfig.java

```diff
@@ -123,13 +125,13 @@ public void validateTable(String tableName, SourceValueType valueType, FailureCo
                             String tableField) {
     // Call API to fetch first record from the table
     ServiceNowTableAPIRequestBuilder requestBuilder = new ServiceNowTableAPIRequestBuilder(
-      connection.getRestApiEndpoint(), tableName, false)
+      connection.getRestApiEndpoint(), tableName, false, SchemaType.SCHEMA_API_BASED)
```

Comment:
Why is this always SCHEMA_API ? 

What happens in case of 1.1 ( string based ) ?

Reply:
this is not a schema call, as previous parameter is showing false. Thats why we are setting default schema type